pykilosort/postprocess.py

```diff
@@ -322,7 +322,12 @@ def ccg(st1, st2, nbins, tbin):
     #                      - vary for different setups.
     st1 = cp.asnumpy(st1)
     st2 = cp.asnumpy(st2)
-    return _ccg(st1, st2, nbins, tbin)
+    try:
+        return _ccg(st1, st2, nbins, tbin)
+    except ValueError:
+        print(st1.shape)
```

Comment:
logger.exception

Reply:
This was probably a temporary debug print statement which i forgot to remove. I'll remove it in a future commit.